frontend/redux/selectors.js

```diff
@@ -24,3 +24,22 @@ export const isLoading = state => Object
 
 export const demographics = state => state.demographics.questions;
 export const demographicsOrder = state => state.demographics.order;
+
+export const selectWorkload = function(state) {
+  const images = [];
+  const answers = store.getState().demographics.answers;
```

Comment:
This function is a "selector" it should be able to select the values needed from an arbitrary object representing a state.